backend/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSubscriptionTypeProcessor.java

```diff
@@ -103,158 +74,84 @@
-                });
-    }
+    private Multi<NotificationHistory> oldSendEmail(Action action, EmailSubscriptionType emailSubscriptionType, EmailTemplate emailTemplate) {
+        return subscriptionResources
+                .getEmailSubscribersUserId(action.getAccountId(), action.getBundle(), action.getApplication(), emailSubscriptionType)
+                .onItem().transform(Set::copyOf)
+                .onItem().transformToMulti(users -> {
+                    if (!emailTemplate.isSupported(action.getEventType(), emailSubscriptionType)) {
+                        return Multi.createFrom().empty();
+                    }
 
-    private Uni<String> renderTitle(Notification item, EmailSubscriptionType emailSubscriptionType, EmailTemplate emailTemplate) {
-        return emailTemplate.getTitle(item.getAction().getEventType(), emailSubscriptionType)
-                .data("action", item.getAction())
-                .createMulti()
-                .collect().with(Collectors.joining())
-                .onFailure()
-                .recoverWithItem(templateEx -> {
-                    log.log(Level.WARNING, templateEx, () -> String.format(
-                            "Unable to render template title for application: [%s], eventType: [%s], subscriptionType: [%s].",
-                            item.getAction().getApplication(),
-                            item.getAction().getEventType(),
-                            emailSubscriptionType
-                    ));
-                    return null;
-                });
-    }
+                    TemplateInstance subject = emailTemplate.getTitle(action.getEventType(), emailSubscriptionType);
+                    TemplateInstance body = emailTemplate.getBody(action.getEventType(), emailSubscriptionType);
 
-    private Uni<Tuple3<Email, String, String>> combine(Email email, Uni<String> title, Uni<String> body) {
-        return Uni.combine().all()
-                .unis(
-                        Uni.createFrom().item(email),
-                        title,
-                        body
-                ).asTuple()
-                .onItem().transform(objects -> {
-                    if (objects == null || objects.getItem1() == null || objects.getItem2() == null || objects.getItem3() == null) {
-                        return null;
+                    if (subject == null || body == null) {
+                        return Multi.createFrom().empty();
                     }
 
-                    return objects;
+                    return emailSender.oldSendEmail(users, action, subject, body).toMulti();
                 });
     }
 
+    /*
+    * Warning: This method is not in sync with the current daily-aggregation as a cron job in the backend.
+    * It still sends a single email (as opposed to one email per user) with the users in the bcc
+    * This is going to be updated in a separated PR to match the personalized email.
+    * */
     @Incoming(AGGREGATION_CHANNEL)
```

Comment:
@gwenneg Adding `oldSendEmail et al` methods to support this flow temporally. I discussed with @bennetelli  and we are going to fix it on a following PR.